shared/models/check_cancelation.yml

```diff
@@ -0,0 +1,11 @@
+description: >-
```

Comment:
NITPICK: could you change the name of this file to `check_cancellation`? Cancelation isn't incorrect, but it's also very very rare compared to the double-L spelling.